phpcs.xml.dist

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Well, I would argue we should add more rules than core, because unlike core, we can easily fullfill more of them, like for example short array syntax and what not. This is really a minimal configuration